src/unity/python/turicreate/toolkits/image_similarity/image_similarity.py

```diff
@@ -201,6 +214,9 @@ def _load_version(cls, state, version):
         _tkutl._model_version_check(version, cls._PYTHON_IMAGE_SIMILARITY_VERSION)
         from turicreate.toolkits.nearest_neighbors import NearestNeighborsModel
         state['similarity_model'] = NearestNeighborsModel(state['similarity_model'])
+        if state['model'] == "VisionFeaturePrint_Screen" and _mac_ver() < (10,14):
+            raise ToolkitError("Can not load model on this operating system. This model uses VisionFeaturePrint_Screen, " \
```

Comment:
Nit: Note that `\` is not necessary inside parens.

Reply:
No need to change, just an FYI.

Reply:
Interesting. I didn't know that. I updated here and in the image classifier.